.ci/validation.sh

```diff
@@ -262,6 +262,22 @@ verify-no-exception-configs)
   exit $fail
   ;;
 
+verify-regexp-id)
+  cd config
+  RESULT=0
+  for FILE in *_checks.xml
```

Comment:
we can specify directory explicitly here: `for FILE in config/*_checks.xml` without `cd`, what do you think?

Reply:
You're right. This looks more elegant.